tarantula.lic

```diff
@@ -39,10 +40,19 @@ class Tarantula
     UserVars.tarantula_last_skillset = @last
   end
 
+  def in_combat?
+    @combat_scripts.each do |script|
```

Comment:
I think you could simplify this to `@combat_scripts.any? { |script| Script.running?(script) }`?

Reply:
Do you know about this `running?` method in lich.rbw? Looks like it might take a list but has some other conditions as well. Could be much simpler: 

https://github.com/dragon-realms/dr-lich/blob/master/lich.rbw#L9790

Reply:
Interesting, yeah it works as expected afaict:
``` 
  1 scripts = ["banana", "potato", "dependency"]
  2 running?(scripts) ? echo("#{scripts} -- something in here is running") : echo("nothing in the list is running"
```

Reply:
Interesting. Based on a quick read of the code in that function, I would have expected it to require that ALL of the scripts in the list are running.

There are some interesting edge cases with the way that works today:
```
R>;la
--- Lich: dependency, roomnumbers, drinfomon, afk, spellmonitor, smartlisten, attention, sanowret-crystal, skill-recorder, moonwatch, t2, events, jail-buddy, hunting-buddy, textsubs, combat-trainer, lnet, log

>;e echo running?(['notcombat-trainer'])
[exec1: true]

R>;e echo running?(['foo', 'fi', 'bar'])
[exec1: true]
```

Reply:
good eye, the regex match is definitely not what we want. i will probably go back to the recommendation from @rpherbig